test/host/socket/wasi_socket.cpp

```diff
@@ -16,6 +17,8 @@
 #include <string>
 #include <string_view>
 
+#define SWAP_E(Val) WasmEdge::EndianValue(Val).le()
+
```

Comment:
What is the reason we need to add this MACRO? Why don't we just use the original statement?